src/components/NavBar/NavMenu.jsx

```diff
@@ -86,7 +93,7 @@ const NavMenu = ({
             >
               <MenuItem
                 component={Button}
-                startIcon={<ContactsIcon />}
+                startIcon={<ContactsIcon style={{ height: '24px', width: '24px' }} />}
```

Comment:
For MUI components, think we could utilize their specialized `sx` prop for custom styling instead of `style` since we're not using HTML elements.